tests/openvino/test_quantization.py

```diff
@@ -75,7 +75,7 @@
 class OVQuantizerTest(unittest.TestCase):
     SUPPORTED_ARCHITECTURES_WITH_EXPECTED_QUANTIZED_MATMULS = (
         (OVModelForSequenceClassification, "bert", 32, 35),
-        # (OVModelForCausalLM, "gpt2", 41, 23),
+        (OVModelForCausalLM, "gpt2", 41, 3),
```

Comment:
I don't feel like the number int8 constants are correct in this test. @nikita-savelyevv, can you please take a look?

Reply:
@AlexKoff88, when running quantization on OV backend, the number of int8 constants is 22. For PT backend it's only 3 though. I think there are some issues on PT backend, will look into it or create a ticket.